bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,694 @@
+		<label>Show WLAN status with LED</label>
+	</channel-type>
+
+	<channel-type id="TimeServer" advanced="true">
+		<item-type>String</item-type>
+		<label>Timeserver</label>
+	</channel-type>
+
+	<channel-type id="geopos" advanced="false">
+		<item-type>Location</item-type>
+		<label>Location of air-Q Device</label>
+	</channel-type>
+
+	<channel-type id="nightmode_StartDay" advanced="false">
+		<item-type>String</item-type>
+		<label>Start of Day Operation</label>
+	</channel-type>
+
+	<channel-type id="nightmode_StartNight" advanced="false">
+		<item-type>String</item-type>
+		<label>End of Day Operation</label>
+	</channel-type>
+
+	<channel-type id="nightmode_BrightnessDay" advanced="false">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Day Brightness of LED</label>
+	</channel-type>
+
+	<channel-type id="nightmode_BrightnessNight" advanced="false">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Night Brightness of LED</label>
+	</channel-type>
+
+	<channel-type id="nightmode_FanNightOff" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Switch off fan at Night</label>
+	</channel-type>
+
+	<channel-type id="nightmode_WifiNightOff" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Switch off WLAN at Night</label>
+	</channel-type>
+
+	<channel-type id="devicename" advanced="false">
+		<item-type>String</item-type>
+		<label>Device Name</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="RoomType" advanced="false">
+		<item-type>String</item-type>
+		<label>Room Type</label>
+	</channel-type>
+
+	<channel-type id="Logging" advanced="false">
+		<item-type>String</item-type>
+		<label>Logging level</label>
+	</channel-type>
+
+	<channel-type id="DeleteKey" advanced="false">
+		<item-type>String</item-type>
+		<label>Settings to be Deleted</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="FireAlarm" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Fire Alarm</label>
+	</channel-type>
+
+	<channel-type id="WLAN_config_Gateway" advanced="false">
+		<item-type>String</item-type>
+		<label>Network Gateway</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="WLAN_config_MAC" advanced="false">
+		<item-type>String</item-type>
+		<label>MAC Address</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="WLAN_config_SSID" advanced="false">
+		<item-type>String</item-type>
+		<label>WLAN SSID</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="WLAN_config_IPAddress" advanced="false">
+		<item-type>String</item-type>
+		<label>Assigned IP Address</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="WLAN_config_NetMask" advanced="false">
+		<item-type>String</item-type>
+		<label>Network Mask</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="WLAN_config_BSSID" advanced="false">
```

Comment:
Consider making any channels that a lot of users will not want to link to an item as advanced="true"
The more channels that are linked, the more work the openHAB core has to do so it is an advantage to encourage users to not link everything.

Reply:
I changed many more channels to 'advanced'. One question in this context: It would be great if there were a state `writeonly="true"`, but it seems OH does not have that. The air-Q has some parameters that should have a channel where a user can write to, but don't give back a value, e.g. the Log Level or the password. Is there a more elegant way to implement these than just having a channel providing the value NULL?

Reply:
That would be best asked on the forum in the developer area as my knowledge is limited. I suspect the answer will be that you just tell the main UI to not display the state. There are also thing ACTIONS that you can implement into a binding which could match what your after in some cases perhaps?

https://www.openhab.org/docs/developer/bindings/#actions-bound-to-a-thing